etcd/conf.yaml

```
@@ -176,8 +176,8 @@ eduid:
             AUTH_SHARED_SECRET: shared_secret_shared_secret_sh4r
             DEFAULT_FINISH_URL: https://www.eduid.se/
             # These keys are for dashboard.eduid.docker:8080
```

I suppose this comment may be a bit confusing now (I am taking a guess, mainly to test this new github feature "insert a suggestion" =) )
```suggestion
            # These keys are for signup.eduid.docker:8080
```

---

I agree :)